### DISCUSSION TITLE
Integrated Ice Extent Error (IIEE)

### DISCUSSION THREAD

**Discussion by rgrumbine** [dtcenter/METplus#2316] [Statistical Computation] [answered]
Posting this in support of our tele-discussion, wherein it seemed likely to be easy to add the capability.

Namely, we would like to be able to take a model grid of ice cover and an observed grid of ice cover and compute the total area that either the model has ice but obs does not, or obs has ice but the model does not.

An important detail is that users need to be able to control the critical level for 'has ice'. 15/40/80% are mentioned in the 2021 metrics workshop. To that should be added epsilon (any value > 0). There is a known misbehavior where the ice model can produce extensive areas of low concentration ice.

Also important, the model and observation grids will not have the same land mask. Differences due to land mask should be ignored.

While it seems likely to be easy to do this with existing mode tools, for reference here is the original paper developing IIEE:
https://agupubs.onlinelibrary.wiley.com/doi/full/10.1002/2015GL067232

**Comment by JohnHalleyGotway** [comment, marked as answer]
@rgrumbine thanks for adding these references. As mentioned in #2314, I do think we should identify a METplus scientist to collaborate with you on these details. I'll note that @AliciaBentley-NOAA added the dtcenter/MET#2658 issue to state the need for METplus to read tripolar data directly without using xESMF. And that functionality is a necessary pre-cursor to progress in this area.

From my perspective, the ultimate goal is developing a METplus use case to demonstrate this IIEE workflow.

Regarding differing definitions of land, presumably any grid point containing land could be marked as containing "bad data". When doing grid-to-grid comparisons in MET, if either the forecast or observation contains bad data then that grid point is excluded from the analysis. That approach should apply well in this case.

I agree that the MODE tool serves as a nice example of this type of functionality. One detail to note though is that MODE reports areas as counts of grid boxes rather than true grid box areas. If true grid areas are needed, that discrepancy would need to be addressed.

**Reply by rgrumbine** [reply]
The 'bad data' solution sounds good to me.

Unfortunately, yes, true area is needed.

**Comment by j-opatz** [comment]
At today's METplus User Telecon, this issue was discussed. I've turned this into an issue where we can keep track of further developments and any acceptance criteria: https://github.com/dtcenter/MET/issues/2887

With the issue creation I'll also close this Discussion.
